### ISSUE TITLE
TypeError: can't pickle lxml.etree.QName objects

### ISSUE THREAD

**Issue by kurtrwall** [mvantellingen/python-zeep#842]
Hello,

I'm trying to send SOAP headers along with my request, but I'm getting a pickle error.

```python
# zeep 3.1.0
from zeep import Client
client = Client(wsdl='https://datafeeds.symantec.com/Feeds/Datafeed.asmx?WSDL')
authHeader = client.get_element('ns0:AuthHeader')
authHeader.UserName = '***'
authHeader.Password = '***'

# below raises "TypeError: can't pickle lxml.etree.QName objects"
response = client.service.GetFeedFileList(25, -1, _soapheaders=[authHeader])
```

**Comment by sgang007** [comment]
Can you display the output of `type(authHeader)`

**Comment by mvantellingen** [comment]
I'm closing this issue due to the low quality of the initial report. You are welcome to create a new issue but please make sure it contains all the relevant information and that it is properly formatted.

Also please remember that this project is opensource and completely unpaid work.

**Comment by kurtrwall** [comment]
The issue was opened with the requested information at the time of report. It seems that the template for opening issues on this repository has been changed.

I ended up using a different library because of this limitation before I opened the issue. I had hoped the report would be beneficial to the folks using and working on this, even though I had no personal stake in it. Good luck with the project!
